cmd/fleet/server_test.go

```diff
@@ -46,7 +47,7 @@ func TestRunServer(t *testing.T) {
 	et, err := NewEnrollerT(verCon, cfg, nil, c)
 	require.NoError(t, err)
 
-	router := NewRouter(ctx, bulker, ct, et, nil, nil, nil, nil)
+	router := NewRouter(ctx, bulker, ct, et, nil, nil, nil, nil, fbuild.Info{})
 	errCh := make(chan error)
 
 	var wg sync.WaitGroup
```

Comment:
@aleksmaus Can you add a test to assert format response, this would prevent us future breaking changes.